Classes/EventSourcedRepository.php

```diff
@@ -69,9 +69,7 @@ public function findByIdentifier($identifier)
             throw new AggregateRootNotFoundException(sprintf("Could not reconstitute the aggregate root %s because its class '%s' does not exist.", $identifier, $this->aggregateClassName), 1474454928115);
         }
 
-        $aggregateRootReflection = new \ReflectionClass($this->aggregateClassName);
-        /** @var EventSourcedAggregateRootInterface $aggregateRoot */
-        $aggregateRoot = $aggregateRootReflection->newInstanceWithoutConstructor();
+        $aggregateRoot = unserialize('O:' . strlen($this->aggregateClassName) . ':"' . $this->aggregateClassName . '":0:{};');
```

Comment:
Fix a regression, don't check why but @bwaidelich fix that last week ... look a wrong merge happen at some point.